controller_manager:
  ros__parameters:
    update_rate: 250 # [Hz] - Set 250 Hz for 4 ms or 83 Hz for 12 ms cycle time

    defaults:
      allow_controller_activation_with_inactive_hardware: true
      deactivate_controllers_on_hardware_self_deactivate: false

    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    control_mode_handler:
      type: kuka_controllers/ControlModeHandler
    event_broadcaster:
      type: kuka_controllers/EventBroadcaster
    kss_message_handler:
      type: kuka_controllers/KssMessageHandler
    gpio_controller:
      type: gpio_controllers/GpioCommandController
